frontend/packages/knative-plugin/src/components/revisions/RevisionList.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Table } from '@console/internal/components/factory';
+import RevisionHeader from './RevisionHeader';
+import RevisionRow from './RevisionRow';
+import { RevisionModel } from '../../models';
+
+const RevisionList: React.FC = (props) => (
```

Comment:
Can you create an `interface` for the props here?

Reply:
Changed to use `TableProps`.